src/data/causeSpecificJSONData/cats.json

```diff
@@ -0,0 +1,45 @@
+{
+  "path": "cats",
+  "data": {
+    "styles": {
+      "colors": {
+        "primary": "FF6A08",
+        "secondary": "29BEBA",
+        "secondaryShadow": "15AAA6",
+        "background": "FAEFE1"
+      }
+    },
+    "sections": {
+      "landing": {
+        "title": "Open tabs.  Help shelter cats!",
+        "subtitle": "Every tab opened raises money for cats in need of an extra, helping paw. It’s so easy a cat could do it.",
+        "ctaImg": "cat1.png"
```

Comment:
I'm thinking we can just pass down the file name of the images, and then we can lazy load them inside the react component.  Cursory googling makes it seem like this is very doable